GameServer/Logic/RAF/RAFManager.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace LeagueSandbox.GameServer.Core.Logic.RAF
 {
-    static class RAFManager
```

Comment:
This is still pretty much a static class, since all the members are static. I'd like this to work the same way as the old RAFManager did with Ninject, but in case you don't feel like doing that now, I don't blame you.